packages/@react-types/card/src/index.d.ts

```diff
@@ -22,13 +23,17 @@ interface SpectrumCardProps extends AriaCardProps, StyleProps, DOMProps {
   layout?: 'grid' | 'waterfall' | 'gallery',
   // TODO: readd size when we get updated designs from spectrum
   // size?: 'S' | 'M' | 'L',
-  orientation?: 'horizontal' | 'vertical'
+  orientation?: Orientation
 }
 
 interface LayoutOptions {
   // cardSize?: 'S' | 'M' | 'L',
-  // cardOrientation?: 'horizontal' | 'vertical',
-  collator?: Intl.Collator
+  cardOrientation?: Orientation,
+  collator?: Intl.Collator,
+  // TODO: is this valid or is scale a spectrum specific thing that should be left out of the layouts?
```

Comment:
scale is spectrum specific, but i think our Layouts are all Spectrum specific? so should be fine?